app/views/scripts/production_deploy.erb

```diff
@@ -25,6 +25,12 @@ if [ "${HEROKU_APPS}" == "" ]; then
   exit 1
 fi
 
+if [ "${USE_HEROKU_DEPLOY_WRAP}" == "1" ]; then
+  GIT_PUSH="heroku-deploy-wrap -- git push"
```

Comment:
Just question, how can we use [heroku-deploy-wrap](https://github.com/yuya-takeyama/heroku-deploy-wrap) in deploy-support-tools? I mean installation of heroku-deploy-wrap.